### DISCUSSION TITLE
How to submit gene/variant to matchmaker exchange?

### DISCUSSION THREAD

**Discussion by AlessandroLas** [broadinstitute/seqr#3029] [General]
How to submit gene/variant to matchmaker exchange?

The video seqr: Matchmaker Exchange (at https://www.youtube.com/watch?v=DTYjbGgPe8M) 
shows that variants with MME tag could be retrieved in the matchmaker section of the project/sample.

However, I couldn't find such list of variants for submission.

Thanks in advance.

**Comment by hanars** [comment]
Hi Alessandro,

In order to submit to matchmaker, you need to first have tagged variants of interest in the family of interest. Any variant you have tagged in a specific family is eligible for submission, it does not matter what the tag you use is. Follow the instructions in the video starting at 0:45 for instructions on how to submit a new case to matchmaker

**Reply by AlessandroLas** [reply]
Hi hana.

Thanks for the reply.
Following the instructions, I first tagged my variant of interest. Then, from the family page (this is a single sample with no pedigree),  when I click on the MatchMaker Exchange link, I can only see the "Add note" button.
By contrast, everything works fine with a real family trio.
Is this a wanted feature of the tool?

Best.

**Reply by hanars** [reply]
Please provide a link to the matchmaker page where you are experiencing the issue so I can better understand the issue

Also, please bear in mind that matchmaker is a production system connected to other institutions with which the Broad has a professional relationship. If you have an actual gene of interest that you want to connect with researchers about, you should proceed with using matchmaker. However, if you are just experimenting with what is available in the seqr system, please don't. It is unfair to the researchers who will be contacted about spurious matches

**Reply by AlessandroLas** [reply]
Thanks Hana.

I solved. 

For single samples I couldn't see the list of variants to submit to matchmaker until I added sex and affected status in the "Edit Individuals" tab.
For family trios this was automatically set when I loaded the "individuals" file ("Bulk edit individuals") listing the individuals and their relationships.

As of now I'm experimenting with the features of seqr system on local install (in view of a possible implementation at my institute) and I am perfectly aware of the reasons why the community was created for.
